table/block_based_table_reader.cc

```diff
@@ -2406,6 +2407,28 @@ Status BlockBasedTable::Get(const ReadOptions& read_options, const Slice& key,
           break;
         }
 
+        bool hash_effective, found;
+        biter.SeekForGet(key, &hash_effective, &found);
+
+        if (hash_effective) {
+          if (found) {
+            ParsedInternalKey parsed_key;
+            s = Status::OK();
```

Comment:
You don't explicitly need to set the status here. We implicitly assume s is ok() until it is checked.